src/js/sw/index.js

```diff
@@ -62,7 +62,7 @@ async function handleFontRequest(request) {
 addEventListener('fetch', event => {
   const url = new URL(event.request.url);
 
-  if (url.host == 'fonts.gstatic.com') {
+  if (url.pathname.endsWith('.woff2')) {
```

Comment:
Just wondering, does it make sense to keep the separate font cache?

Reply:
It does for the latin-ext and vietnamese font, else they'll be lost in version bumps

Reply:
Do we even need to treat those differently? I mean, we only have latin characters :)

Reply:
Ah, true, thanks!